miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolOrderRestController.java

```diff
@@ -157,4 +162,32 @@ private static PoolOrderDto writeUrls(PoolOrderDto poolOrderDto, UriComponentsBu
     return poolOrderDto;
   }
 
+  @RequestMapping(value = "/poolorder/picker/active", method = RequestMethod.GET, produces = { "application/json" })
+  @ResponseBody
+  public PoolPickerResponse getPickersByUnfulfilled(@RequestParam("platform") String platform) throws IOException {
+    return getPoolPickerWithFilters(100,
+        PaginationFilter.platformType(PlatformType.get(platform)),
+        PaginationFilter.fulfilled(false));
+  }
+
+  @RequestMapping(value = "/poolorder/picker/chemistry", method = RequestMethod.GET, produces = { "application/json" })
+  @ResponseBody
+  public PoolPickerResponse getPickersByChemistry(@RequestParam("platform") String platform,
+      @RequestParam("seqParamsId") Long paramsId,
+      @RequestParam("fulfilled") boolean fulfilled) throws IOException {
+    return getPoolPickerWithFilters(100,
+        PaginationFilter.platformType(PlatformType.get(platform)),
+        PaginationFilter.fulfilled(fulfilled),
+        PaginationFilter.sequencingParameters(paramsId));
+  }
+
+  private PoolPickerResponse getPoolPickerWithFilters(Integer limit, PaginationFilter... filters) throws IOException {
+    PoolPickerResponse ppr = new PoolPickerResponse();
+    ppr.populate(poolOrderCompletionService, true, "lastUpdated", limit, PoolOrderRestController::orderTransform, filters);
+    return ppr;
+  }
+
+  private static PoolPickerEntry orderTransform(PoolOrderCompletion order) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)